docs/konnectors.md

```diff
@@ -320,25 +320,57 @@ On the host create the file /etc/rkt/net.d/10-containers.conf
 
 and run your container with the "--net=bridge" option. That way, a new interface is available in the container and gives you access to the host.
 
+## Konnector install and run details
+
+### Install
+
+The konnectors will be installed in the .cozy_konnectors directory which is in the VFS using git clone
+(like the apps at the moment).
+
+The konnectors installation may be triggered when the user says he wants to use it. The resulting
+repository is then kept for each run of the konnector. It may then be given to the user the
+possibility to upgrade the konnector to the latest version if any.
+
+To update a given konnector, a ```git pull``` command is run on the konnector.
+
+### Details about running a konnector
+
+To run a given konnector, the stack will copy this connector in a "run" directory, which is not in
+the VFS. This directory will be given to the rocket container as the current working directory
+with full read and write access on it. This is where the container will put its logs and any temp
+file needed. There will be also cozy-client.js and the shared libraries in a lib directory inside
+this directory. The lib directory will be the content of the [actual server lib
+directory](https://github.com/cozy-labs/konnectors/tree/master/server/lib).
+
+The konnector will be run with the following environment variables :
+
+- COZY_TOKEN
+- COZY_URL : to know what instance is running the konnector
+- KONNECTOR_CREDENTIALS : as a json string with all the values from the account associated to the
+konnector.
+
+In the end of the konnector execution (or timeout), the logs are read in the log.txt file and added
```

Comment:
If rocket has something aking to docker logs, it might be easier to just use the process stdout.